gms/feature_service.hh

```diff
@@ -108,6 +108,7 @@ public:
     gms::feature tombstone_gc_options { *this, "TOMBSTONE_GC_OPTIONS"sv };
     gms::feature parallelized_aggregation { *this, "PARALLELIZED_AGGREGATION"sv };
     gms::feature keyspace_storage_options { *this, "KEYSPACE_STORAGE_OPTIONS"sv };
+    gms::feature typed_errors_in_read_rpc { *this, "TYPED_ERRS_IN_READ_RPC"sv };
```

Comment:
s/ERRS/ERRORS/ please. There's no reason to abbreviate, and it's inviting confusion when someone does a `git grep -i typed_errs_in_read_rpc` without realizing it's different.

Reply:
I abbreviated it because dtests look for the "ERROR" string in the logs on teardown. The feature name is printed to the logs when the node starts and most tests failed. Here is a CI run which failed due to this reason: https://jenkins.scylladb.com/job/releng/job/Scylla-CI/92/

I'll send a PR to scylla-ccm to improve the error detection in the logs. Maybe we can also change the name of the feature and align the variable name with the string - e.g. `TYPED_EXCEPTIONS_IN_READ_RPC`?

Reply:
I'm okay with either name.